File: exs/mundo_1/python/014.py
"""
Desafio 014

Problema: Escreva um programa que converta uma temperatura
          digitando em graus Celsius e converta para graus Fahrenheit.

Resolução do problema:
"""
temperatura_C = float(input('Informe a temperatura em °C: '))

temperatura_F = 1.8 * temperatura_C + 32

print('Temperatura em: {:.1f}°F'.format(temperatura_F))

File: exs/mundo_1/c#/014.cs
/*
Desafio 014

Problema: Conversor de temperaturas: escreva um programa que converta uma temperatura 
          digitada em ºC para ºF

Resolução do problema:
*/

using System;

static class io {
    public static string input(string text) {
        Console.Write(text);
        return Console.ReadLine();
    }
    
    public static void print(object data, string end="\n") {
        string text = data.ToString();

        Console.Write($"{text}{end}");
    }
}

class Program {
    public static void Main() {
        double temperatura_C = double.Parse(
          io.input("Informe a temperatura em °C: ")
        );

        double temperatura_F =  1.8 * temperatura_C + 32;

        io.print($"Temperatura: {temperatura_F}°F");
    }
}